File: mssideimss-acuses/src/main/java/mx/gob/imss/acuses/dto/PlantillaDatoDto.java
package mx.gob.imss.acuses.dto;

import java.io.Serializable;
import java.util.Map;

import lombok.Data;
import mx.gob.imss.acuses.enums.TipoAcuse;

@Data
public class PlantillaDatoDto implements Serializable {

    private static final long serialVersionUID = 1L;

    private Long cveIdPlantillaDatos;
    private String nomDocumento;
    private String desVersion; // Corresponde a desVersion de PlantillaDato
    private String datosJson; // Aquí se pasará la cadena JSON
    private TipoAcuse tipoAcuse;

    private Map<String, Object> additionalParameters;

    public PlantillaDatoDto() {
    }


}

File: guisideimss_portal/src/app/business/contador/model/PlantillaDatoDto.ts
export interface PlantillaDatoDto {
  cveIdPlantillaDatos: number | null;  
  nomDocumento: string;
  desVersion: string; // Corresponde a desVersion de PlantillaDato
  datosJson: string; // Aquí se pasará la cadena JSON
  tipoAcuse: string;
}